server/src/test/java/com/broadcom/lsp/cobol/usecases/TestSameCopybookStatementsInDifferentPlacesTreatedAsDifferentEntries.java

```diff
@@ -18,36 +18,26 @@
 import com.broadcom.lsp.cobol.positive.CobolText;
 import com.broadcom.lsp.cobol.usecases.engine.UseCaseEngine;
 import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.DiagnosticSeverity;
 import org.junit.jupiter.api.Test;
 
 import java.util.List;
 import java.util.Map;
 
 import static com.broadcom.lsp.cobol.service.delegates.validations.SourceInfoLevels.ERROR;
-import static com.broadcom.lsp.cobol.service.delegates.validations.SourceInfoLevels.WARNING;
-import static org.eclipse.lsp4j.DiagnosticSeverity.Error;
-import static org.eclipse.lsp4j.DiagnosticSeverity.Warning;
 
 /**
  * Test several copy statements treated as different entries, so if one of them contains a syntax
- * errors, it will be shown only on the copy statement that produces the error. Here: one of two
- * "COPY REPL" statements places in the procedure division, that is, according to its content, a
- * syntax error, so this exact statement should be underlined.
+ * errors, it will be shown only on the copy statement that produces the error. Here: duplicated
+ * DATA DIVISION is a syntax error, but it is shown only on the second copybook entry.
  */
 class TestSameCopybookStatementsInDifferentPlacesTreatedAsDifferentEntries {
   private static final String TEXT =
       "       IDENTIFICATION DIVISION.\n"
           + "       PROGRAM-ID. TESTREPL.\n"
-          + "       DATA DIVISION.\n"
-          + "       WORKING-STORAGE SECTION.\n"
-          + "       01  {$*PARENT}.\n"
           + "       COPY {~REPL}.\n"
-          + "       PROCEDURE DIVISION.\n"
-          + "       {#*MAINLINE}. \n"
-          + "           MOVE 0 TO {$TAG-ID}.\n"
-          + "           MOVE {_COPY {~REPL}.|1|2|3_}\n"
-          + "           GOBACK. ";
-  private static final String REPL = "       {05|1} {$*TAG-ID|2|3} PIC 9.\n";
+          + "       {_COPY {~REPL}.|1_}\n";
+  private static final String REPL = "       {DATA|1} DIVISION.\n";
```

Comment:
You have copied this REPL twice in your program, but only the second copy contains an error. That means that you use the same `1` diagnostic tag in two places. I expect that the test fails, but it is not.

Reply:
A-ha! I got it—the error related to copybook, not extended document. Now I see that it's OK.

Reply:
Yes, that's correct. The reason why I changed this test was just that we are going to support a lot of new semantic features, so most likely the number of errors, as well as their messages, will change frequently. Here I just check that the different copy statements represent different copybook entries, so the test semantic remains the same after the changes.